qa/system-indices/build.gradle

```diff
@@ -23,3 +23,5 @@ testClusters.configureEach {
   setting 'xpack.security.autoconfiguration.enabled', 'false'
   user username: 'rest_user', password: 'rest-user-password'
 }
+
+tasks.named('test').configure { enabled = false}
```

Comment:
Why is this necessary?

Reply:
there were no tests we did this in other places. But you're right we should just rely on gradle skipping those tasks anyhow.

Reply:
removed